changelogs/fragments/7746-raw_post-without-actions.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - xcc_redfish_command - added support for raw_post without a specific action info (https://github.com/ansible-collections/community.general/pull/7746).
```

The name `raw_post` is not exposed to users of the module. They have to select `category=Raw` and `command=PostResource`.
```suggestion
  - xcc_redfish_command - added support for raw POSTs (``command=PostResource`` in ``category=Raw``) without a specific action info (https://github.com/ansible-collections/community.general/pull/7746).
```